idle_sweep.go

```diff
@@ -119,11 +119,10 @@ func (is *idleSweep) checkIdleConnections() {
 			continue
 		}
 
-		is.ch.log.WithFields(
-			LogField{"remotePeer", conn.remotePeerInfo},
```

Comment:
Looks like we lose `remotePeerInfo.Version` when we get rid of this since it's not naturally logged as part of the Connection's logger, https://github.com/uber/tchannel-go/blob/dev/connection.go#L310.  Do we care about that?

Reply:
We add the remote peer info a little later,
https://github.com/uber/tchannel-go/blob/dev/connection.go#L390

Reply:
Ah, okay. 👍